clang/lib/AST/Expr.cpp

```diff
@@ -3183,10 +3183,24 @@ Expr *Expr::IgnoreUnlessSpelledInSource() {
     }
     return E;
   };
+
+  auto IgnoreImplicitCallSingleStep = [](Expr *E) {
+    if (auto *C = dyn_cast<CallExpr>(E)) {
+      auto NumArgs = C->getNumArgs();
+      if (NumArgs == 1 ||
+          (NumArgs > 1 && isa<CXXDefaultArgExpr>(C->getArg(1)))) {
+        Expr *A = C->getArg(0);
+        if (A->getSourceRange() == E->getSourceRange())
+          return A;
+      }
+    }
+    return E;
+  };
+
```

Comment:
Can you add a comment explaining it's used for std::get/structured binding?

Reply:
done